src/SDK/SmartDetectors.Extensions/Presentation/AlertExtensions.cs

```diff
@@ -194,25 +220,11 @@ private static AlertProperty CreateAlertProperty(Alert alert, AlertPresentationP
                         ConvertChartAxisTypeToContractsChartType(chartAttribute.YAxisType),
                         listValues.Select(point => new ContractsChartPoint(point.X, point.Y)).ToList());
 
-                case AlertPresentationLongTextAttribute longTextAttribute:
-                    return new LongTextAlertProprety(propertyName, displayName, presentationAttribute.Order, PropertyValueToString(propertyValue));
+                case AlertPresentationLongTextAttribute _:
```

Comment:
> _ [](start = 55, length = 2)

Why do you need the "_" ? Same below...

Reply:
that's the C# pattern, tried without it - got compilation error

---
In reply to: [232753804](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/22#discussion_r232753804) [](ancestors = 232753804)